packages/adblocker-puppeteer/adblocker.ts

```diff
@@ -226,6 +226,10 @@ export class PuppeteerBlocker extends FiltersEngine {
 
           // Allows to get styles for updated DOM.
           getRulesFromDOM: true,
+
+          reference: {
+            frame,
```

Comment:
does frame have an `.id`? would prefer to pass it instead of full object. again, we don't want to retain reference to it as it may cause memory leaks

Reply:
No, I don't think so. I just dropped this part.